perennialAutodidact/memory-snap
# serve react app from express

﻿## Description of changes 

- Created a `serve` command in the root-level `package.json` and in the frontend/backend `package.json` files. It builds the frontend application and then starts the backend server which serves the production build of the application. This is in order to avoid running two servers in production on [Render](https://render.com)

## Test steps
- Run `yarn start` in the root directory, the local frontend and backend servers should start and everything should work as expected. 
- Run `yarn serve` in the root directory. The frontend package's `build` command should be run and the backend package's `serve` command should be run in order to serve the production build of the frontend package
